packages/core/saved-objects/docs/openapi/entrypoint.yaml

```diff
@@ -0,0 +1,56 @@
+openapi: 3.1.0
+info:
+  title: Saved objects
+  description: OpenAPI schema for saved object endpoints
+  version: '0.1'
+  contact:
+    name: Kibana Core Team
+  license:
+    name: Elastic License 2.0
+    url: https://www.elastic.co/licensing/elastic-license
+tags:
+  - name: saved objects
+    description: Manage Kibana saved objects, including dashboards, visualizations, and more.
+servers:
+  - url: 'http://localhost:5601'
+    description: local
+paths:
+# Paths in the default space
```

Comment:
Yes for now they're commented out and will be uncommented in other PRs rather than all at once.
It's also worth noting that we're only including the `/api/...` paths at this point, not the `/s/<space>/api/...` paths. If both will be supported in serverless, we'll need to add thsoe as well.